docs/src/components/Layout.tsx

```diff
@@ -26,18 +26,25 @@ const StyledMain = styled.main`
 interface Props {
   children: React.ReactNode;
   location: WindowLocation;
+  title: string;
+  description?: string;
 }
 
-export default function Layout({ children, location }: Props) {
+export default function Layout({ children, location, title, description }: Props) {
   return (
-    <ThemeProvider theme={defaultTheme}>
-      <DevModeProvider>
+    <>
+      <Helmet>
+        <meta charSet="utf-8" />
+        <title>{title}</title>
```

Comment:
Seems like the site title should go here, rather than be part of the titles of individual documents. Can make an exception for home page.